contracts/gateway/src/contract.rs

```diff
@@ -115,8 +119,9 @@ pub mod execute {
             msg: to_binary(&aggregate_verifier::msg::ExecuteMsg::VerifyMessages {
                 messages: unverified
                     .into_iter()
-                    .map(connection_router::msg::Message::from)
-                    .collect(),
+                    .map(|m| m.clone().try_into())
```

Comment:
```suggestion
                    .map(|m| m.try_into())
```

Reply:
it's going to get deleted in the next PR anyway, so ignoring it here 😄